cli/src/pcluster/resources/head_node/user_data.sh

```diff
@@ -70,8 +70,6 @@ if [ "${!custom_cookbook}" != "NONE" ]; then
   else
     cookbook_url=${!custom_cookbook}
   fi
-else
```

Comment:
since we don't have runtime baking, we can remove this line in compute node user data too.

Reply:
Done